.github/workflows/build_and_test.yml

```diff
@@ -252,7 +252,7 @@ jobs:
     - name: Install Python packages (Python 3.8)
       if: (contains(matrix.modules, 'sql') && !contains(matrix.modules, 'sql-'))
       run: |
-        python3.8 -m pip install 'numpy>=1.20.0' 'pyarrow<5.0.0' pandas scipy xmlrunner
+        python3.8 -m pip install 'numpy>=1.20.0' 'pyarrow<7.0.0' pandas scipy xmlrunner
```

Comment:
This is used for SQL related tests, e.g., in https://github.com/apache/spark/tree/master/sql/core/src/test/resources/sql-tests/inputs/udf